Australia update

Tjscott97 opened the pull request:
Focus tree expansion across economic, army, navy and airforce branches adding 14 new focuses.

Approx. 20 economic ideas merged into one dynamic modifier.

New evolving army idea.

New navy idea.

Minor state resource changes.

Commonwealth Aircraft Corp MIO modified to be able to produce CAS as well as a new unique trait.

Several less valuable focuses across political branch have been shortned and 'Control the ABC' focus has been moved to the shared commie-fascist branch giving a 5% boost to either fascism or communism depending on what branch has been taken.

Multiple focus icons have been updated to more fitting newer icons that have been added to the game since focus tree was made.

Tjscott97 committed:
Australia update

Combines approx. 20 economic ideas into 1 dynamic modifier.

New navy idea.

A few less useful focuses made shorter to 50 days from 70.

New rubber focus, based on historical effort to make rubber plantations in South Australia. https://pir.sa.gov.au/aghistory/industries/other_industries/guayule

New CAS MIO.

Steel production moved from Northern Territory and Victoria to New South Wales, where all steel production was done at the time. https://www.hardbakka.com.au/component/virtuemart/steel-manufacturing-in-australia-a-brief-history

Tjscott97 committed:
Naru phosphate

On lose or recapture of Naru, Australia loses 10% or gains 10% resource gain efficiency.

Tjscott97 committed:
Naru phosphate

On lose or recapture of Naru, Australia loses 10% or gains 10% resource gain efficiency.

Tjscott97 committed:
Focus GFX fix

Applied the correct ANZAC GFX for ANZAC focus

Tjscott97 committed:
Merge branch 'Beta-Edits' of https://github.com/Tjscott97/1956_beta into Beta-Edits

marijn211 commented on history/states/520-Northern Territory.txt:
```diff
@@ -3,7 +3,7 @@ state={
 	name="STATE_520"
 	resources={
-		steel = 22
+		#steel = 22 #there was no steel production in northern territory at this time
```
Could maybe add to the comment that you moved the production to 517

marijn211 commented on history/states/517-Victoria.txt:
```diff
@@ -3,7 +3,7 @@ state={
 	name="STATE_517"
 	resources={
-		steel = 8
+		#steel = 8 #there was no steel production in victoria at this time
```
Idem

marijn211 commented on common/military_industrial_organization/organizations/AST_organization.txt:
```diff
@@ -132,6 +132,23 @@ AST_com_aircraft_corporation_organization = { #
 	}	
 }
 
+AST_R56_com_aircraft_corporation_cas_organization = { #Historically built CAS aircraft
```
Adding a company twice because they built multiple types isn't consistent, try to find a type/modify the upgrade tree so the MIO can apply to both types they produced

marijn211 commented:
A few things are broken, to detect surface-level stuff, launch the game with -debug launch argument and you'll get a debug log with errors

marijn211 reviewed (changes requested):
See above

Tjscott97 committed:
Removed unused scripted effects

Tjscott97 committed:
Merged Commonwealth Aircraft Corp

Merged Commonwealth Aircraft Corporation MIOs and added a unique trait for CAS aircraft to MIO.

Tjscott97 committed:
Control ABC focus + Rubber research

Control ABC focus moved to commie-fascist shared branch, now gives +5% communism or fascism depending on branch taken.
Synthetic rubber research bonus added to rubber focus.

Tjscott97 committed:
Update 517-Victoria.txt

Tjscott97 committed:
Update 520-Northern Territory.txt

Tjscott97 committed:
Economic focus branch rework

5 new focuses added to the economic branch of focus tree, 2 focuses from air force branch of tree has been moved over to economic tree where i feel they fit better (uranium mines and nuclear research).

A couple of existing economic focuses have been made shorter, from 70 days to 35 as their rewards were very little.  There are still a few 70 day economic focuses that I feel don't reward enough to justify being 70 days, would love some additional feedback on this, whether they should be shortened, or rewards improved.

Tjscott97 committed:
Focus icon update

Updated multiple focus icons with newer more fitting icons that already exist in game files.

Tjscott97 committed:
Naval focus branch rework

5 new focuses add to naval branch of focus tree.

Rewards have been added to a couple existing focuses, including ships being added to the production queue.

A couple focuses that have lackluster rewards have been shortened.

Tjscott97 committed:
Minor Air focus rework

2 additional focuses added, paratrooper focus moved over from army branch.

A few focuses have been shortened or rewards increased.

Tjscott97 committed:
Minor army focus rework

2 new focuses

Rewards have been added to a couple existing focuses including a new evolving Australian army idea.

Some focuses have been shortened.

Tjscott97 committed:
Political focuses shortened

Several political focuses across the communist and fascists branches that provided little to no rewards have been shortened.

marijn211 committed:
Merge branch 'Beta-Edits' into pr/3307

marijn211 committed:
Slightly reorganize

marijn211 commented on common/on_actions/r56_on_actions.txt:
```diff
@@ -21367,6 +21370,28 @@ on_actions = {
+			if = {
+				limit = {
+					FROM = { tag = AST } 
+					NOT = { ROOT = { tag = AST } }
```
If you want to exclude civil wars, use original_tag instead of tag, FROM and ROOT can't both be the tag AST

marijn211 commented:
Diff and error log look good aside from that thing above, would only need a glance in-game and then should be good

Tjscott97 committed:
Small rewards increase for Labor gov focus tree

Very small rewards increase, +10% line retention for and +10% monthly pop for Nationalize Healthcare

Tjscott97 committed:
Requested fix

Tjscott97 committed:
Merge branch 'Beta-Edits' of https://github.com/Tjscott97/1956_beta into Beta-Edits

Tjscott97 committed:
Delete australia_on_actions.txt

marijn211 committed:
Remove unused on_action

marijn211 committed:
Revert "Remove unused on_action"

This reverts commit 2349209bdf48541c949ad51f64b0742fe16fb194.

marijn211 committed:
Fix on_action

marijn211 committed:
Clean up some indentation

marijn211 committed:
PR-unrelated tooltip improvement

marijn211 commented:
Please indent properly in future code

marijn211 committed:
Much cleaner tooltip for a few focuses

Tjscott97 committed:
Updated tooltips

Tjscott97 committed:
Doctrine cost reduction fix

Correctly applied marines focus to give marines doctrine cost reduction instead of paratrooper doctrine.

Tjscott97 committed:
Army focus cost reduction

Reduced the completion time of 2 not very valuable army focuses.

Tjscott97 committed:
small cost reduction

Tjscott97 committed:
Small cost reductions for economic tree

3 focuses have had small cost reductions.

Tjscott97 committed:
Update australia.txt

research boost added to focus

Tjscott97 committed:
Update r56_british_monarchy_in_exile.txt

moved focus branch over to the right as to not cover 'end white Australia' focus when choosing the fascist branch of tree.

marijn211 committed:
Clean up indents and one excess newline tooltip
